scipy/signal/__init__.py

```diff
@@ -326,6 +327,7 @@
 from ._lti_conversion import *
 from ._signaltools import *
 from ._savitzky_golay import savgol_coeffs, savgol_filter
+from .wigner import wigner_ville
 from ._spectral_py import *
```

Comment:
Fixed this in 14bdfbb.

Reply:
Thank you @tupui for your remarks. I almost considered this PR to be dead. 
I hope I have resolved all your remarks accordingly. There seem to have been quite some changed to the general conventions since I opened the PR.